io.github.cxong.cdogs-sdl.desktop

```
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=C-Dogs SDL
+GenericName=Action/Arcade Game
+Version=0.7.2
```

```suggestion
Version=0.7.2
```
The version here is for the Spec version and not the app version.